mindsdb/integrations/postgres_handler/postgres_handler.py

```diff
@@ -0,0 +1,101 @@
+from mindsdb.integrations.libs.base_handler import DatabaseHandler
+import psycopg
+from mindsdb.utilities.log import log
+from contextlib import closing
+from mindsdb_sql import parse_sql
+
+class PostgresHandler(DatabaseHandler):
+    """
+    This handler handles connection and execution of the PostgreSQL statements. 
+    """
+
+    def __init__(self, name, **kwargs):
+        super().__init__(name)
+        self.parser = parse_sql
+        self.dialect = 'postgresql'
+        self.database = kwargs.get('database')
+        self.host = kwargs.get('host')
+        self.port = kwargs.get('port')
+        self.user = kwargs.get('user')
+        self.password = kwargs.get('password')
+
+    def __connect(self):
+        """
+        Handles the connection to a PostgreSQL database insance.
+        """
+        #TODO: Check psycopg_pool
+        connection = psycopg.connect(f'host={self.host} port={self.port} dbname={self.database} user={self.user} password={self.password}', connect_timeout=10)
+        return connection
+
+    def check_status(self):
+        """
+        Check the connection of the PostgreSQL database
+        """
+        connected = False
+        try:   
+            con = self.__connect()
+            with closing(con) as con:
+                with con.cursor() as cur:
+                    cur.execute('select 1;')
+            connected = True
+        except Exception as e:
+            log.error(f'Error connecting to PostgreSQL {self.database} on {self.host}!')
+            pass
+        return connected
+    
+    def run_native_query(self, query_str):
+        """
+        Receive SQL query and runs it
+        :param query_str: The SQL query to run in PostgreSQL
+        :return: returns the records from the current recordset
+        """
+        con = self.__connect()
+        with closing(con) as con:
+            with con.cursor() as cur:
+                res = True
+                try:
+                    cur.execute(query_str)
+                    res = cur.fetchall()
+                except psycopg.Error as e:
+                    log.error(f'Error running {query_str} on {self.database}!')
+                    pass
+        return res
```

Comment:
May define response same as here: https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/api/http/namespaces/sql.py#L12
So we can run queriess with and without result useing one method.
Also, will be good to output `column_types` for `table` response